packages/app/src/overview/components/update-os-modals/update-available-modal/update-available-modal.component.tsx

```diff
@@ -28,38 +53,68 @@ const messages = defineMessages({
   },
 })
 
+const ModalContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  > p {
+    text-align: center;
+    line-height: 1.4;
+    white-space: pre-wrap;
+  }
+`
+
 export const UpdateAvailableModal: FunctionComponent<
   UpdateAvailableModalProps
-> = ({ onDownload, version, date, open, onClose }) => (
-  <OSUpdateModal
-    open={open}
-    closeButton={false}
-    closeable
-    actionButtonLabel={intl.formatMessage(messages.updateAvailableButton)}
-    onActionButtonClick={onDownload}
-    closeModal={onClose}
-  >
-    <RoundIconWrapper>
-      <Icon type={IconType.Pure} width={3.2} />
-    </RoundIconWrapper>
-    <ModalMainText
-      displayStyle={TextDisplayStyle.Headline4}
-      message={messages.updateAvailableMessage}
-    />
-    <Text
-      displayStyle={TextDisplayStyle.Paragraph4}
-      color="secondary"
-      message={{
-        ...messages.updateAvailableDescription,
-        values: {
-          version,
-          date: new Date(date).toLocaleDateString("en-US", {
-            year: "numeric",
-            month: "short",
-            day: "numeric",
-          }),
-        },
-      }}
-    />
-  </OSUpdateModal>
-)
+> = ({ open = false, releases, onClose, onDownload }) => {
+  return (
+    <Modal
+      isOpen={open}
+      style={getModalDialogStyle({ size: ModalSize.Medium })}
+      shouldCloseOnOverlayClick={false}
+      onAfterClose={onClose}
+    >
+      <Header
+        titleOrder={TitleOrder.TitleFirst}
+        subtitleGap={false}
+        data-testid={ModalTestIds.Header}
+      >
+        <ModalTitle
+          displayStyle={getTitleStyle(ModalSize.Small)}
+          element={"h2"}
+          data-testid={ModalTestIds.Title}
+          message={messages.muditaOsUpdateTitle}
+        />
+        <Close
+          displayStyle={DisplayStyle.IconOnly}
+          Icon={IconType.Close}
+          data-testid={ModalTestIds.CloseButton}
+        />
+      </Header>
+      <ModalContent>
+        <RoundIconWrapper>
+          <Icon type={IconType.Pure} width={3.2} />
+        </RoundIconWrapper>
+        <ModalMainText
+          displayStyle={TextDisplayStyle.Headline4}
+          message={messages.updateAvailableMessage}
+        />
+        <Text
+          displayStyle={TextDisplayStyle.Paragraph4}
+          color="secondary"
+          message={messages.updateAvailableDescription}
```

Comment:
Similar to one of the comments above, but according to the design the title might be in singular/plural form (`You can download the latest MuditaOS version.` and `You can download all the MuditaOS versions available to you with one click.`)

Reply:
https://github.com/mudita/mudita-center/pull/1049/commits/b1e89e36ec965f086ebdbf036cf35ef7965c500d